src/controllers/submissionController.js

```diff
@@ -0,0 +1,85 @@
+const submissionQueries = require('../db/queries.submission.js');
+const mongoose = require('mongoose');
+const Mailgun = require('mailgun').Mailgun;
+const mg = new Mailgun(process.env.mgApi); //Need Mailgun API key
+
+module.exports = {
+  index(req, res, next) {
+    submissionQueries.getAllSubmissions()
+    .then((submissions) => {
+      let returnData = {
+        statusCode: 200,
+        message: 'Success',
+        data: submissions
+      };
+      res.json(returnData)
+    })
+    .catch((err) => {
+      console.log(err)
+      let returnData = {
+        statusCode: 400,
```

Comment:
Probably a `500` rather than a `400`, unless we know the request data is improper

Reply:
Understood, we can add a validation layer before to return 400 errors if needed.